Makefile

```diff
@@ -48,6 +48,7 @@ install: bin bin/$(BINARY_NAME)_ui
 # Clean up images and binaries
 clean:
 	rm -f bin/wego pkg/flux/bin/flux
+	rm -f bin/wego pkg/flux/bin/fluxchecksums.txt
```

Comment:
This will try and delete bin/wego again. 
I think it reads better to put the flux (bin, checksum) removal on their own line.